paasta_tools/cli/cmds/status.py

```diff
@@ -655,6 +655,11 @@ def format_kubernetes_pod_table(pods):
             health_check_status = PaastaColors.grey("N/A")
         elif pod.phase == "Running":
             health_check_status = PaastaColors.green("Healthy")
+            for container in pod.containers:
+                # a container is waiting or terminated
+                if container.tail_lines.error_message:
+                    health_check_status = PaastaColors.red("Unhealthy")
+                    break
```

Comment:
I sort of worry this would cause the inconsistent view between paasta status and the replication check. Could you make [is_pod_ready](https://sourcegraph.yelpcorp.com/mirrors/Yelp/paasta/-/blob/paasta_tools/kubernetes_tools.py#L1623) as the only function to check if the pod is healthy.